idaes/generic_models/properties/core/examples/tests/test_BT_PR.py

```diff
@@ -176,9 +176,9 @@ def test_T350_P1_x5(self, m):
         assert pytest.approx(
                 value(m.fs.state[1].enth_mol_phase["Vap"]), 1e-5) == 78048.7
         assert pytest.approx(
-                value(m.fs.state[1].entr_mol_phase["Liq"]), 1e-5) == -367.558
+                value(m.fs.state[1].entr_mol_phase["Liq"]), 1e-5) == -361.794
```

Comment:
There are two different things that we might want to test:
1) Has our implementation changed?
2) Are the values accurate?

These sorts of checks validate (1) but not (2). There are certainly properties that we *know* give inaccurate results (see my struggles with water VLE) just from modeling using a cubic EoS.

The best bet would be to validate pure component properties against a high-fidelity EoS used by NIST or whoever. Unfortunately, this would not have been a property we could have tested against. However, we could look into creating those tests in the future, probably for hydrocarbons properties and VLE, which Cubics model really well.

If we started including experimental data, we would have to deal with the further complication of experimental error. (Not to mention it's impossible to measure entropy directly.)

If I have some extra time, I'll try to look into limiting cases for cubics. We know as pressure drops to zero they should behave like an ideal gas---we could create a battery of tests comparing them to the Ideal property package at low pressures. Even as the pressure increases some, I think we should be able to compare a second virial coefficient against some other gases.

Reply:
Added a new test featuring comparisons to CoolProp's implementation of cubic EoS  and values of partial molar quantities calculated using a cubic EoS from a paper. I'd say that most of our methods now have at least some independent validation, but there was a bit of mismatch between the partial molar enthalpies calculated using our equations and those included in that paper. CoolProp doesn't provide partial molar quantities, so it cannot serve as a tiebreaker.